pkg/repositories/transformers/node_execution_test.go

```diff
@@ -167,6 +166,61 @@ func TestCreateNodeExecutionModel(t *testing.T) {
 	}, nodeExecutionModel)
 }
 
+func TestCreateNodeExecutionModelStartNode(t *testing.T) {
+	nodeExecutionModel, err := CreateNodeExecutionModel(ToNodeExecutionModelInput{
+		Request: &admin.NodeExecutionEventRequest{
+			Event: &event.NodeExecutionEvent{
+				Id: &core.NodeExecutionIdentifier{
+					NodeId: "start-node",
+					ExecutionId: &core.WorkflowExecutionIdentifier{
+						Project: "project",
+						Domain:  "domain",
+						Name:    "name",
+					},
+				},
+				Phase: core.NodeExecution_RUNNING,
+				OutputResult: &event.NodeExecutionEvent_OutputUri{
+					OutputUri: "output uri",
+				},
+				OccurredAt: occurredAtProto,
+				ParentTaskMetadata: &event.ParentTaskExecutionMetadata{
+					Id: &core.TaskExecutionIdentifier{
+						RetryAttempt: 1,
+					},
+				},
+			},
+		},
+		ParentTaskExecutionID: 8,
+	})
+	assert.Nil(t, err)
+
+	var closure = &admin.NodeExecutionClosure{
```

Comment:
shouldn't the output result from the event also be captured in this closure?

Reply:
Removed this test as falling back to not fetching the data on start node input if the InputUri is not set